tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -77,3 +78,10 @@ Wait Until Build Status Is
     [Arguments]    ${namespace}    ${build_name}    ${expected_status}=Complete    ${timeout}=20 min
     Wait Until Keyword Succeeds    ${timeout}    1 min
     ...    Build Status Should Be    ${namespace}    ${build_name}    ${expected_status}
+
+Wait Until All Builds Are Complete
+    [Documentation]     Waits until all the builds are in Complete State
+    ${builds_data} =  Oc Get  kind=Build  namespace=redhat-ods-applications
```

Comment:
Please, add a `${namespace}` parameter instead of having redhat-ods-applications in the keyword

Reply:
Updated